cosmic_ray/operators/number_replacer.py

```diff
@@ -6,14 +6,25 @@
 from .operator import Operator
 
 
+# Maps mutation count to number offsets
+OFFSET_MAP = {
```

Comment:
Why can't this be a list, which is indexed by integers starting from zero anyway? If it's helpful to explicitly see the indexes, I'm happy with it being a dictionary.

Reply:
Good idea! The dict is certainly overkill for this.